explode/dpkg.go

```diff
@@ -26,7 +26,7 @@ import (
 // Dpkg will explode all .deb's specified and then return
 // the install/ path inside that exploded tree.
 func Dpkg(pkgs []string) (string, error) {
-	rootDir, err := ioutil.TempDir("", "abireport-dpkg")
+	rootDir, err := ioutil.TempDir("/var/tmp", "abireport-rpm")
```

Comment:
Because this is in the dpkg handling code, I think it would be better to keep the second argument unmodified, or else use a generic string like `"abireport"`.